### PR TITLE
remove old unavailable link with relevant fix patch 1

### PR DESCRIPTION
**Pull request by Gaurav2728**

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb [hunk 1]
```diff
@@ -915,7 +914,7 @@ def configure_connection
         variables['wait_timeout'] = self.class.type_cast_config_to_integer(wait_timeout)
 
         # Make MySQL reject illegal values rather than truncating or blanking them, see
-        # http://dev.mysql.com/doc/refman/5.6/en/server-sql-mode.html#sqlmode_strict_all_tables
+        # http://dev.mysql.com/doc/refman/5.6/en/sql-mode.html#sqlmode_strict_all_tables
```
Comment (carlosantoniodasilva) [inline activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb]: This is an unrelated change to removing the dev.rubyonrails.org links, can you please do it on a separate commit instead?

Comment (Gaurav2728) [inline activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb]: @carlosantoniodasilva yes, previous link is unavailable or removed. I do it in separate PR as you suggest.

**Comment by Gaurav2728** [comment]
@carlosantoniodasilva PR is updated and as your suggestion i send seperate PR for replaced mysql link https://github.com/rails/rails/pull/19469

**Comment by carlosantoniodasilva** [comment]
Thanks :heart:
